ui/apos/components/AposExportModal.vue

```diff
@@ -407,7 +388,7 @@ export default {
   flex-direction: column;
   gap: 20px;
   align-items: baseline;
-  height: auto;
+  height: calc(100% - 60px); // IMPORTANT: 60px is the sum of the margin-bottom, the gap and the margin-top of .apos-export__settings
```

Comment:
we use 100% here so that a long translation for the following selected text would still display correct heights:

<img width="502" alt="image" src="https://github.com/user-attachments/assets/c4ba3607-dc6c-4e20-94bb-2bff79cd0b72" />